packages/image_picker/image_picker/example/ios/Runner.xcodeproj/project.pbxproj

```diff
@@ -407,6 +475,11 @@
 			target = 97C146ED1CF9000F007C117D /* Runner */;
 			targetProxy = 6800491C2280D368006DD6AB /* PBXContainerItemProxy */;
 		};
+		6801C83C2555D726009DAF8D /* PBXTargetDependency */ = {
```

Comment:
This says that the test target depends on the Runner (app) target, so Runner needs to be built first.